app/views/planning_application/review_policy_classes/edit.html.erb

```diff
@@ -0,0 +1,130 @@
+              </p>
+              <% if comment = policy_form.object.comment.presence %>
+                <h4 class="govuk-heading-s policy-comment-title">
+                  <%= existing_policy_comment_label(comment) %>
+                </h4>
+                <p class="govuk-body"><%= simple_format(comment.text) %></p>
+              <% end %>
+            </td>
+            <% Policy.statuses.each_key do |status| %>
+              <td class="govuk-table__cell">
+                <div class="govuk-form-group">
+                  <fieldset class="govuk-fieldset">
+                    <div class="govuk-radios" data-module="govuk-radios">
+                      <div class="govuk-radios govuk-radios--small">
+                        <div class="govuk-radios__item">
+                          <%= policy_form.radio_button(
+                                :status,
+                                status,
+                                class: "govuk-radios__input",
+                                disabled: @planning_application.assessment_complete?
+                              ) %>
+                          <%= policy_form.label(
+                                :status,
+                                "&nbsp;".html_safe,
+                                class: "govuk-label govuk-radios__label"
+                              ) %>
+                        </div>
+                      </div>
+                    </div>
+                  </fieldset>
+                </div>
+              </td>
+            <% end %>
+          </tr>
+        <% end %>
+        </tbody>
+      </table>
+      <%= form.fields_for :review_policy_class, @policy_class.review_policy_class do |review_form| %>
+        <div class="govuk-form-group <%= review_form.object.errors.any? ? 'govuk-form-group--error' : '' %>">
+          <fieldset class="govuk-fieldset">
+            <% if review_form.object.errors.any? %>
+              <% review_form.object.errors.each do |error| %>
+                <p id="status-error" class="govuk-error-message">
+                  <span class="govuk-visually-hidden">Error:</span><%= error.message %>
+                </p>
+              <% end %>
+            <% end %>
+
+            <div class="govuk-radios" data-module="govuk-radios">
+              <%= review_form.govuk_radio_button(:mark, :accept, label: { text: "Accept" }) %>
+              <%= review_form.govuk_radio_button(
+                :mark, :return_to_officer_with_comment, label: { text: "Return to officer with comment" }
+              ) do %>
+                <%= review_form.govuk_text_area(
+                      :comment,
+                      label: { text: "Explain to the assessor why this needs reviewing" },
+                      rows: 6
+                    ) %>
+              <% end %>
+            </div>
+          </fieldset>
+        </div>
+      <% end %>
+      <div class="govuk-button-group">
```

Comment:
I think we have a partial for this...`"shared/submit_buttons"`

Reply:
Fixed this and all other examples I could find in the application.